Exceptions/src/com/company/IntNum.java

```diff
@@ -0,0 +1,29 @@
+package com.company;
+import com.company.Exceptions.InvalidNumberException;
+import com.company.Exceptions.NonNumberException;
+
+import java.util.Scanner;
+
+public class IntNum {
+    private static int MinValidNumber = 1;
+    private static int MaxValidNumber = 100;
+    private static Scanner scanner = new Scanner(System.in);
+    private static int number;
+
+    static void validateNumber() throws InvalidNumberException, NonNumberException {
+        System.out.println("Please enter the integer number in the range [1...100]:");
+        String input = null;
+        try {
+            input = scanner.nextLine();
+            number = Integer.parseInt(input);
+        } catch (Exception exception) {
```

Comment:
catch particular exceptions, not general one

Reply:
fixed